githooks/pre-commit

```diff
@@ -4,7 +4,6 @@ STAGED_GO_FILES=$(git diff --cached --name-only | grep ".go$")
 
 for FILE in ${STAGED_GO_FILES}
 do
-    gofmt -w -s "${FILE}"
```

Comment:
❤️

Reply:
goimports is needed because some imports have to be removed after generation to avoid compilation errors (e.g. `io`)